internal/pkg/gateway/api.go

```diff
@@ -279,31 +280,46 @@ func (gs *Server) Submit(ctx context.Context, request *gp.SubmitRequest) (*gp.Su
 		return nil, status.Errorf(codes.Unavailable, "no orderer nodes available")
 	}
 
-	orderer := orderers[0] // send to first orderer for now
+	// randomly shuffle orderers
+	rand.Shuffle(len(orderers), func(i, j int) { orderers[i], orderers[j] = orderers[j], orderers[i] })
```

Comment:
Instead of shuffling perhaps use [rand.Perm](https://pkg.go.dev/math/rand#Perm) ? It's shorter syntax and conveys better what we want to accomplish

Reply:
Yes, that's definitely neater.  Thanks.